dev-deployment/deploy-javabuilder-dev-with-ssl.sh

```diff
@@ -0,0 +1,148 @@
+#!/bin/bash
+
+# Deploy JavaBuilder with SSL certificates for dev environment  
+# Based on production buildspec.yml using existing wildcard certificate
+
+set -e
+
+PROFILE="codeorg-dev"
+REGION="us-east-1"
+STACK_NAME="javabuilder-dev"
+TEMPLATE_PATH="../cicd/3-app/javabuilder"
+APP_TEMPLATE="app-template.yml"
+PACKAGED_TEMPLATE="packaged-app-template.yml"
+
+# Set artifact bucket - use environment variable if available, otherwise use default
+if [ -z "$ARTIFACT_STORE" ]; then
+    ARTIFACT_STORE="javabuilder-dev-artifacts"
+fi
+# Check if artifact bucket exists, create if needed
+echo "🔍 Checking if artifact bucket exists: $ARTIFACT_STORE"
+if ! aws s3api head-bucket --bucket "$ARTIFACT_STORE" --profile "$PROFILE" --region "$REGION" 2>/dev/null; then
+    echo "📦 Creating artifact bucket: $ARTIFACT_STORE"
+    aws s3 mb "s3://$ARTIFACT_STORE" --profile "$PROFILE" --region "$REGION"
+else
+    echo "✅ Artifact bucket already exists: $ARTIFACT_STORE"
+fi
+
+# Ensure Java is in PATH  
+export PATH="/opt/homebrew/opt/openjdk@11/bin:$PATH"
```

Comment:
This is specific to an engineer using a macOS development environment. Might be simpler to document in a README what the JDK prerequisite is.